README.md

```diff
@@ -1,25 +1,58 @@
+<img src="https://raw.githubusercontent.com/libplctag/libplctag.NET/master/assets/libplctag-final.svg?sanitize=true" alt="libplctag" width="180"/>
+<p>
+
 # libplctag .NET
 
+## Packages
 This repository contains two .NET packages for [libplctag](https://github.com/libplctag/libplctag) that are published to Nuget.org:
-* [libplctag](https://www.nuget.org/packages/libplctag/) ![Nuget version](https://img.shields.io/nuget/vpre/libplctag) ![Nuget](https://img.shields.io/nuget/dt/libplctag)
-* [libplctag.NativeImport](https://www.nuget.org/packages/libplctag.NativeImport/) ![Nuget version](https://img.shields.io/nuget/vpre/libplctag.NativeImport) ![Nuget](https://img.shields.io/nuget/dt/libplctag.NativeImport)
 
-### libplctag
+| Package | Downloads | Stable | Preview |
+|-|-|-|-|
+| [libplctag](https://www.nuget.org/packages/libplctag/) | ![Nuget](https://img.shields.io/nuget/dt/libplctag) | ![Nuget version](https://img.shields.io/nuget/v/libplctag) | ![Nuget version](https://img.shields.io/nuget/vpre/libplctag) |
+| [libplctag.NativeImport](https://www.nuget.org/packages/libplctag.NativeImport/) | ![Nuget](https://img.shields.io/nuget/dt/libplctag.NativeImport) | ![Nuget version](https://img.shields.io/nuget/v/libplctag.NativeImport) | ![Nuget version](https://img.shields.io/nuget/vpre/libplctag.NativeImport) |
+
+
+## libplctag
+
+This is the package intended for use in .NET applications. It depends on libplctag.NativeImport to gain access to the underlying libplctag native library.
 
-This is the package intended for use in .NET applications.
 It provides an API for libplctag that should feel natural to .NET developers by supporting the following features:
 * Values are strongly-typed (both Atomic types and User-Defined Types).
 * Errors are thrown as Exceptions
 * Async/Await
 * Native resource cleanup
 
+
+## libplctag.NativeImport
+
+Most developers will not need to directly reference the Native Import library. This library automatically extracts platform-specific version of the base libplctag library needed for the libplctag .NET wrapper. 
+
+If you wish to override this behavior you can do so: [Using a non packaged version of the native libplctag library](https://github.com/libplctag/libplctag.NET/wiki/Using-a-non-packaged-version-of-the-native-libplctag-library)
```

Comment:
I'd prefer that we keep this in readme because the wiki doesn't support Pull Requests - is that OK? 
We can add a second readme, or a documentation folder full of markdown files with documentation content which works the same as the wiki + has Pull Request functionality.

Reply:
The main wiki for the core library is accessible at libplctag.wiki IIRC.   That said, I do not see a way in the GitHub UI to make a PR.   A little searching doesn't turn anything up.  Huh.  That seems like a big miss on GitHub's part.

Looks like most people adopt the doc directory idea you had.  Maybe I should change my wiki too?

Reply:
I actually like wikis for documentation - they have a nice table of contents, they're searchable, etc. It is weird that the wiki isn't in the actual code repo - that's dumb.

My thought on the Native Import example/documentation was that it didn't belong in the main README. I think we've seen a couple of cases where people have tried to use it just because they saw it in the README and didn't read closely enough to understand it wasn't necessary.

I was also thinking that maybe the README could have a section of "known issues" and include a list of wikis/examples/whatever for things like Immutable Directories, non-default platforms, etc.

I've been doing a decent amount of MQTT stuff and I find myself referencing the [MQTTNet](https://github.com/chkr1011/MQTTnet/wiki) wiki pretty often. We need somewhere that we can have articles/explanations like that for this project. On the other hand, Newtonsoft relies on their [website](https://www.newtonsoft.com/json/help/html/Introduction.htm) for example/documentation. Not sure I have a preference, but I have found that my co-workers just don't seem to dig into the example cs files for whatever reason...

Reply:
Alright, after finding some repos that do a good job with the 'docs' folder full of markdown files I can get on board with that. I'll do some refactor.